content/guidedLearning/videos/RTSGameDev.md

```diff
@@ -7,28 +7,24 @@ further-reading:
 video-overview:
 video-content:
     - title: RTS Game
-      url: https://www.youtube.com/watch?v=i7tdJpf6Pnc&list=PLCrwuqjmVebK8M9DW_MNOxqn5ic6hF4ua&index=1
-    - title: RTS Game Development Series
-      url: https://www.youtube.com/playlist?list=PLCrwuqjmVebK8M9DW_MNOxqn5ic6hF4ua      
+      url: https://www.youtube.com/watch?v=i7tdJpf6Pnc

```

Comment:
what was wrong with the URL that includes the list?

Reply:
As I wrote, it didn't work. That playlist is gone.

Reply:
Very weird. When I tested it before I could see the list (this is why I asked). But now it is gone. All good, thanks a lot :-)